decentraland/explorer-desktop
# feat: custom post processing stack & depth of field setting

## What does this PR change?


Adds a custom post-processing volume for desktop project when initializing the scene.
The new volume contains depth of field, color adjustements & vignette components.
A new setting has been added into graphic category for depth of field

## How to test the changes?

1. Open the launcher in developer mode and set the custom branch to `feat/custom-post-processing-stack`
2. Move around in the world, you should see that distant objects are kind of blurry
3. Press TAB, go to graphic settings and disable Depth of Field. You should check that the blurry efect its gone
4. Restart the app. Depth of Field should remain disabled
5. Open settings and press RESET button, depth of field should be enabled again

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
